tests/gradients/parameter_shift/test_parameter_shift_shot_vec.py

```diff
@@ -2308,9 +2290,7 @@ def test_tf(self, broadcast, tol):
         # assert np.allclose(hess[0][:, 2:5], np.zeros([2, 3, 3]), atol=tol, rtol=0)
         # assert np.allclose(hess[1][:, -1], np.zeros([2, 1, 1]), atol=tol, rtol=0)
 
-    # TODO: Torch support for param-shift
     @pytest.mark.torch
-    @pytest.mark.xfail
```

Comment:
how was this still failing?

Reply:
It was failing because `res` was a tuple and a tuple didn't have a `detach` method, see update made below.